Podfile

```diff
@@ -1,3 +1,4 @@
+source 'https://github.com/CocoaPods/Specs.git'
```

Comment:
I think the CDN was correct, this is old approach ... right?

Reply:
Ah yes, there was an error message when no source is provided saying to add the git source!

Reply:
Oh no, I read it wrong: https://github.com/CocoaPods/CocoaPods/blob/0c940a3097e9eae0c5269f5101ac3276e02e0d33/lib/cocoapods/installer.rb#L772
```ruby
UI.warn 'Your project does not explicitly specify the CocoaPods master specs repo. Since CDN is now used as the default'
```

Will update.